File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithSlidesInPresentation/Transitions.php
<?php

namespace Aspose\Slides\WorkingWithSlidesInPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;
use com\aspose\slides\TransitionType as TransitionType;


class Transitions {

    public static function run($dataDir=null)
    {

        # Instantiate Presentation class that represents the presentation file
        $pres = new Presentation($dataDir . 'demo.pptx');

        $transition_type = new TransitionType();

        # Apply circle type transition on slide 1
        $pres->getSlides()->get_Item(0)->getSlideShowTransition()->setType($transition_type->Circle);

        # Apply comb type transition on slide 2
        $pres->getSlides()->get_Item(1)->getSlideShowTransition()->setType($transition_type->Comb);

        # Saving the presentation
        $save_format = new SaveFormat();
        $pres->save($dataDir . "SimpleTransition.pptx", $save_format->Pptx);

        print "Done with simple transition, please check the output file." . PHP_EOL;

    }
}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithSlidesInPresentation/Transitions.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat
from com.aspose.slides import TransitionType

class Transitions:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWithSlidesInPresentation/Transitions/'    
       
        # Instantiate Presentation class that represents the presentation file
        pres = Presentation(dataDir + 'demo.pptx')

        transition_type = TransitionType

        # Apply circle type transition on slide 1
        pres.getSlides().get_Item(0).getSlideShowTransition().setType(transition_type.Circle)

        # Apply comb type transition on slide 2
        pres.getSlides().get_Item(1).getSlideShowTransition().setType(transition_type.Comb)

        # Saving the presentation
        save_format = SaveFormat
        pres.save(dataDir + "SimpleTransition.pptx", save_format.Pptx)

        print "Done with simple transition, please check the output file."


if __name__ == '__main__':        
    Transitions()